lua/astrocommunity/comment/dooku-nvim/init.lua

```diff
@@ -0,0 +1,10 @@
+return {
+  {
+    "Zeioth/dooku.nvim",
+    cmd = { "DookuGenerate", "DookuOpen", "DookuAutoSetup" },
+    opts = {
+      -- your config options here
+    },
+    config = function(_, opts) require("dooku").setup(opts) end,
```

Comment:
This shouldn't be needed.

Reply:
Fixed in https://github.com/AstroNvim/astrocommunity/pull/503/commits/35134589c4f3b3412ad31a6a1874eddcdf74c897